src/main/java/com/stripe/model/PaymentIntentSourceAction.java

```diff
@@ -8,6 +8,9 @@
 @Setter
 @EqualsAndHashCode(callSuper = false)
 public final class PaymentIntentSourceAction extends StripeObject {
+  PaymentIntentSourceActionValueAuthorizeWithUrl authorizeWithUrl;
   String type;
+
+  // This attribute is deprecated and is here for legacy reasons.
```

Comment:
I can't mark it as really deprecated otherwise the setter used in the deserializer fails